packages/opentelemetry-metrics/src/MeterRegistry.ts

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ConsoleLogger } from '@opentelemetry/core';
+import * as types from '@opentelemetry/types';
+import { Logger } from '@opentelemetry/types';
+import { Meter } from '.';
+import { DEFAULT_CONFIG, MeterConfig } from './types';
+
+/**
+ * This class represents a basic meter registry which platform libraries can extend
+ */
+export class MeterRegistry implements types.MeterRegistry {
+  private readonly _meters: Map<string, Meter> = new Map();
+  readonly logger: Logger;
+
+  constructor(private _config: MeterConfig = DEFAULT_CONFIG) {
+    this.logger = _config.logger || new ConsoleLogger(_config.logLevel);
+  }
+
+  getMeter(
+    name: string = 'default',
```

Comment:
This is same as https://github.com/open-telemetry/opentelemetry-specification/issues/407 and https://github.com/open-telemetry/opentelemetry-js/issues/679, I would suggest to remove default for now and assign the names in examples and tests. We can come back to this and update based on the decision. @xiao-lix and @dyladan WDYT?

Reply:
Definitely `name` should be a required property.

Reply:
Sounds good to me, but I checked out this PR #681 is approved, will it be merged? 🤔

Reply:
unlikely at this point that it will be merged in its current state. in any case, the spec clearly states that name is a required attribute

Reply:
instead of making name required, we are more likely to change the examples to contain better names. Right now many of them are "default" (my fault) which is not a great name and doesn't really help anyone understand the purpose of the name

Reply:
I see, will remove default for now and assign better names in examples and tests.